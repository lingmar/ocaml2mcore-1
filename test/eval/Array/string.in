
let _ =
  let a = Array.make 5 "hatt" in
  print_endline (a.(0));
  Array.iter print_endline a
