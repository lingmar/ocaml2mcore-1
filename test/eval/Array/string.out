hatt
hatt
hatt
hatt
hatt
hatt
